docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -27,7 +27,7 @@ from the top level `alluxio` directory with the following command:
 
 {% include Running-Spark-on-Alluxio/spark-profile-build.md %}
 
-* Add the following line to `spark/conf/spark-env.sh`.
+* Add the following line to `spark/conf/spark-defaults.conf`.
```

Comment:
this works for 2.0.0, 1.6.1 as i tested.

these configurations spark.exeuctor.extraJavaOpts, spark.executor.extraClassPath exists at least since 1.0.0 (http://spark.apache.org/docs/1.0.0/configuration.html)

but spark.driver.extraJavaOpts and spark.driver.extraClassPath are  only added in 1.3.0 . These are useful for local mode only.